routes/stories.js

```diff
@@ -0,0 +1,121 @@
+const express = require('express')
+const router = express.Router()
+const authenticate = require('../middlewares/authenticate')
+const stories = require('../controllers/stories')
+const { authorizeUser } = require('../middlewares/authorization')
+const { createStory, updateStory } = require('../middlewares/validators/stories')
+
+/**
+ * @swagger
+ * /story:
+ *  get:
+ *   summary: Used to get all the stories
+ *   tags:
+ *     - Stories
+ *   responses:
+ *     200:
+ *       description: returns stories
+ *       content:
+ *         application/json:
+ *           schema:
+ *             $ref: '#/components/schemas/stories'
+ *     500:
+ *       description: badImplementation
+ *       content:
+ *         application/json:
+ *           schema:
+ *             $ref: '#/components/schemas/errors/badImplementation'
+ */
+
+router.get('/', stories.fetchStories)
```

Comment:
Can you please add pagination, filters, and sorting?
https://developer.atlassian.com/server/confluence/pagination-in-the-rest-api/
https://jsonapi.org/format/#fetching-sorting

Reply:
Added pagination, filtering, and sorting in fetch stories API